components/sections/Header.jsx

```diff
@@ -1,15 +1,17 @@
 import Wrapper from "../wrappers/Wrapper";
-import Button from "../ui/Button";
+import Link from "next/link";
+import Image from "next/image";
+import ArrowSmall from "../../public/assets/images/arrow_small.svg";
```

Comment:
I suggest you add a more descriptive name to your arrow since we are going to use the arrows from one components